vertx-template-engines/vertx-web-templ-pebble/src/main/java/io/vertx/ext/web/templ/impl/PebbleVertxLoader.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author Paulo Lopes
  */
-class PebbleVertxLoader implements Loader<String> {
+public class PebbleVertxLoader implements Loader<String> {
```

Comment:
We usually expect that users do not use classes directly from any `impl` package since we do not guarantee that the API will be untouched as the project evolves. The issue could arise once we implement java9 jigsaw where the impl packages will not be visible anymore. Since we're not there yet, lets keep it there for now...